CONTRIBUTING.md

```diff
@@ -1,54 +1,65 @@
-# Contributing
+# How to contribute
 
-Issues, content and code contributions are all very appreciated. We ask that you read the relevant sections of this document before contributing.
+Issues, tutorials requests, new tutorials, content fixes and code contributions are all very appreciated. We ask that you read the relevant sections of this document before contributing.
 
+## Contributing a content fix
 
-## Issues
+Have you spotted a typo? A screenshot that needs updating? A nonsensical sentence? While we strive to foster and maintain quality content, mistakes happen. If you have the time, please open a new Pull Request, propose a fix and add the "Tutorials content" label to it, it will be reviewed quickly.
 
-When creating issues, please add as much information as you can and try to split it up into readable sections.
+## Contributing a new tutorial
 
-Some example sections to include:
- - Short summary
- - Steps taken to produce the issue
- - Current and expected result
- - A screenshot to demonstrate the issue
-
-
-## Pull requests
-
-Please try to create simple commits with readable commit messages. Do split the work into smaller pull requests, rather than one big pull request, unless you are proposing a new tutorial.
-
-Some sections to include when creating a PR:
- - Short summary of the work completed
- - Steps to view and test visual or code changes
- - Links to any relevant issues
- - Screenshots to demonstrate the work
+See our [writing guide](https://tutorials.ubuntu.com/?q=tutorial+guidelines) for extensive documentation on writing and publishing a new tutorial. It contains step-by-step documentation, content guidelines and examples to help you make the best of this platform. 
 
+All tutorials are reviewed and merged by the Canonical Docs team in order to maintain content consistency. You can get in touch with us on Freenode IRC, in #ubuntu-doc.
```

Comment:
this is just a suggestion, but I think it helps link to Freenode's webchat when referencing IRC, as readers may be unfamiliar with it, i.e: http://webchat.freenode.net/?channels=ubuntu-doc